### DISCUSSION TITLE
Would Uninstalling Mods or MINT itself Affect my save file?

### DISCUSSION THREAD

**Discussion by Lioju98** [trumank/mint#171] [Q&A] [answered]
Example. I if i download a mod that affects spawn rate for events and cargo crates, if i deleted this mod or Mint itself after claiming these rewards, would it affect my save file and turn it into sandbox?
Sorry if this isn't the appropriate place to post questions

**Comment by trumank** [comment, marked as answer]
The only time a sandbox save file gets created is when explicitly doing so via the sandbox save creation menu when using a sandbox mod through the official integration.

**Comment by jieyouxu** [comment]
Also note that mint itself does not touch your savefile; the mods themselves might.
